src/component/parser/json/converter/CollaborationConverter.ts

```diff
@@ -51,22 +51,24 @@ export default class CollaborationConverter extends AbstractConverter<void> {
   }
 
   private parseCollaboration(collaboration: TCollaboration): void {
-    this.buildParticipant(collaboration['participant']);
-    this.buildMessageFlows(collaboration[FlowKind.MESSAGE_FLOW]);
+    this.buildParticipant(collaboration.participant);
```

Comment:
No related to this PR
I guess this is a better way, no?

Reply:
Yes, I just see it during the refactoring.
It avoids to have a mistake in the name of the property.